core/source/org/libreoffice/ide/eclipse/core/wizards/NewUnoProjectWizard.java

```diff
@@ -363,10 +366,15 @@ protected IStatus run(IProgressMonitor pMonitor) {
             try {
                 IUnoidlProject prj = UnoFactory.createProject(mData, pMonitor);
 
-                mServiceSet.mProject = prj;
-                mServiceSet.doFinish(pMonitor, mActivePage);
+                AbstractLanguage language = (AbstractLanguage) mData.getProperty(IUnoFactoryConstants.PROJECT_LANGUAGE);
 
-                UnoidlProjectHelper.setProjectBuilders(prj);
+                if (!language.getName().contains("Python")) {
```

Comment:
Since restricted for Java Language, XM.idl and M.idl files are not created, since the '.idl' files are not created '.urd' files are also not created resulting which 'types.rdb' is also not created. The MImpl.java is also a file which is not created.

Will look into it in the next version when handling the idl files, and again if condition is I think the only way to block it out.